src/index.js

```diff
@@ -45,6 +30,55 @@ const renderTasks = () => {
           </li>
       `;
     });
+
+  const lists = document.querySelectorAll('li');
+  lists.forEach((list) => {
+    list.addEventListener('click', (e) => {
+      const elem = e.target;
+      if (elem.classList.contains('drag') || elem.classList.contains('check')) {
+        return;
+      }
+
+      lists.forEach((elem) => elem.classList.remove('active'));
+
+      list.classList.add('active');
+
+      const inp = list.querySelector('.input');
+      inp.readOnly = false;
+      inp.focus();
+      inp.setSelectionRange(inp.value.length, inp.value.length);
+    });
+  });
+
+  document.querySelectorAll('li .input').forEach((inp) => {
+    inp.addEventListener('keypress', (e) => {
+      if (e.key === 'Enter') {
+        const id = Number(inp.parentNode.parentNode.id.split('-')[1]);
+        const obj = tasks.find((task) => task.index === id);
+        obj.description = inp.value.trim();
+        updateTask(tasks, obj);
+        inp.readOnly = true;
+      }
+    });
+  });
+
+  document.querySelectorAll('.delete').forEach((delBtn) => {
+    delBtn.addEventListener('click', () => {
+      const id = Number(delBtn.parentNode.parentNode.id.split('-')[1]);
+
+      removeTask(tasks, id);
+      tasks = JSON.parse(localStorage.getItem('tasks'));
+      delBtn.parentNode.parentNode.remove();
+    });
+  });
 };
```

Comment:
- [ ]  Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique.

As you can use on the below image, the indexes are not being updated when the delete function is pressed. so you should update the remaining indexed after deleting a task in the array.

- [ ] When you select the task as completed it doesn't update the local storage, so by default the completed is` false` but you need to update to `true` when the task is done.

![image](https://user-images.githubusercontent.com/55635977/159683795-f6c3fcb5-f405-4746-9578-ef050b095ee8.png)

Reply:
Thank you for feedback.
Added update all remaining items indexes.
Select the task as completed update the local storage to true will be add at last project.